### PR TITLE
title case, resolves #1036

### LINKED ISSUE #1036: Use Title Case on Player Page
* don't display hyphens in file name
* capitalise username
* Title case for title

e.g.  "moon-test-one by matt" becomes "Moon Test One by Matt"

### PR DESCRIPTION
**Pull request by inDigiNeous**

### CODE REVIEW

File: views/graph/show.handlebars [hunk 1]
```diff
@@ -33,6 +33,14 @@
             Share
         </button>
 	</div>
+	<h2 id="graph_heading">
+		<span class="title">{{graphNameHumanReadable}}</span>
+		by
+		<span class="author">{{graphOwnerTitle}}</span>
+	</h2>
+	<div class="progress" style="display:none">
```
Comment (gmarinov) [inline views/graph/show.handlebars]: there shouldn't be a progress div here. (line 11 has the element in question)

File: controllers/graphController.js [hunk 2]
```diff
@@ -141,6 +140,12 @@ GraphController.prototype.embed = function(req, res, next) {
 	}).catch(next)
 }
 
+// "moon-test-one" -> "Moon Test One"
+function convNameToHumanReadable(name) {
```
Comment (gmarinov) [inline controllers/graphController.js]: could we name this somehow better? the method and the parameter?

File: views/graph/show.handlebars (hunk 1 above)
Comment (inDigiNeous) [inline views/graph/show.handlebars]: Yeah got there from a merge conflict, will fix.

File: controllers/graphController.js (hunk 2 above)
Comment (inDigiNeous) [inline controllers/graphController.js]: How ? convGraphNameToHumanReadable(graphName) ?
filterGraphNameForTitle?

What is your preference ? :)

Comment (gmarinov) [inline controllers/graphController.js]: well, this replaces dashes with spaces, so we may decide to use it for something other than names at some other point..

Comment (inDigiNeous) [inline controllers/graphController.js]: It also uppercases every word, so it's kinda specific to the usecase, hence why I named it this way, if we want to change the functionality later, then we would need to change the function name also, don't want to tie too much specifics into any function name.

File: views/graph/show.handlebars [hunk 3]
```diff
@@ -33,6 +33,11 @@
             Share
         </button>
 	</div>
+	<h2 id="graph_heading">
```
Comment (gmarinov) [inline views/graph/show.handlebars]: ermm.. this shouldn't be here? there is one up above already. (assuming this is based on latest develop)

**Comment by kschzt** [comment]
aborted

**Pull request closed without merging**